lib/docusign_dtr/query_param_helper.rb

```diff
@@ -0,0 +1,141 @@
+module DocusignDtr
+  # rubocop:disable Metrics/ClassLength
+  class QueryParamHelper
+    # rubocop:disable Naming/PredicateName
+    QUERY_PARAMS = {
+      search: :search,
+      count: :count,
+      endDate: :end_date,
+      startPosition: :start_position,
+      roomStatus: :room_status,
+      ownedOnly: :owned_only,
+      transactionSide: :transaction_side,
+      isUnderContract: :is_under_contract,
+      regionId: :region_id,
+      officeId: :office_id,
+      hasSubmittedTaskList: :has_submitted_task_list,
+      hasContractAmount: :has_contract_amount,
+      sort: :sort,
+      dateRangeType: :date_range_type,
+      startDate: :start_date
+    }.freeze
+
+    class << self
+      def call(options)
+        @options = options
+        QUERY_PARAMS.each_with_object({}) do |(key, value), memo|
+          memo[key] = send(value) if @options.key? key
+        end
+      end
+
+      def search
+        @options[:search].to_s
+      end
+
+      def count
+        @options[:count].to_i
+      end
+
+      def start_position
+        # binding.pry
+        @options[:startPosition].to_i
+      end
+
+      def room_status
+        raise "error: not acceptable value #{@options[:room_status]}" unless %w[
```

Comment:
do you think it makes sense to move this list of allowed options to the model instead of hardcoding it here?